lib/logstash/outputs/rabbitmq.rb

```diff
@@ -76,20 +71,108 @@ def initialize(params)
     super
   end
 
-  # Use MarchHare on JRuby to avoid IO#select CPU spikes
-  # (see github.com/ruby-amqp/bunny/issues/95).
-  #
-  # On MRI, use Bunny.
-  #
-  # See http://rubybunny.info and http://rubymarchhare.info
-  # for the docs.
-  if RUBY_ENGINE == "jruby"
-    require "logstash/outputs/rabbitmq/march_hare"
+  public
```

Comment:
Sorry, I call bs on this "easier to see" argument. Group your methods and change visibility for every group once (with the "keyword" form, which is just calling a method without any arguments). The entire Ruby community does this.

Not that I expect anything to change because I feel this way :)